src/uct/ib/rdmacm/rdmacm_md.c

```diff
@@ -18,11 +18,23 @@ static ucs_config_field_t uct_rdmacm_md_config_table[] = {
 
   {"ADDR_RESOLVE_TIMEOUT", "500ms",
    "Time to wait for address resolution to complete",
-    ucs_offsetof(uct_rdmacm_md_config_t, addr_resolve_timeout), UCS_CONFIG_TYPE_TIME},
+   ucs_offsetof(uct_rdmacm_md_config_t, addr_resolve_timeout), UCS_CONFIG_TYPE_TIME},
 
   {NULL}
 };
 
+static ucs_config_field_t uct_rdmacm_cm_config_table[] = {
+    {"", "", NULL,
+     ucs_offsetof(uct_rdmacm_cm_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_cm_config_table)},
+
+    {"SOURCE_ADDRESS", "",
+     "Source IPv4 or IPv6 address used in rdma_resolve_addr or empty "
+     "(equivalent to NULL)",
```

Comment:
can the user specify the port to use too? imo need to address this in the comment

Reply:
I think so, limitation are the same as for mentioned  `rdma_resolve_addr`

Reply:
but then if the user sets the port, need to parse it

Reply:
doesn't `inet_pton` do this?

Reply:
not sure it will.
if it will, how should the user provide the port? this param is a string..
if it won't, we either should not support setting the port or not use this function

Reply:
as discussed offline, let's mention in the comment that the port would be selected by rdmacm

Reply:
added